SpeechResponder/Personality.cs

```diff
@@ -123,7 +123,16 @@ public static Personality FromFile(string filename = null, bool isDefault = fals
                 }
                 catch (Exception e)
                 {
-                    Logging.Warn("Failed to access personality at " + filename + ": " + e.Message);
+                    // malformed JSON for some reason: rename so that the user can examine and fix it.
+                    string newFileName = filename + ".malformed";
+                    if (File.Exists(newFileName))
+                    {
+                        // no point keeping a history: only the latest is likely to be useful. Pro users will be using version control anyway.
+                        File.Delete(newFileName);
+                    }
+                    File.Move(filename, newFileName);
+
+                    Logging.Error($"Could not parse \"{filename}\": moved to \"{newFileName}\". Error was \"{e.Message}\"");
```

Comment:
This may be worthy of an audible alert or perhaps a message box to alert the user that the personality has been pruned. Thoughts?

Reply:
As discussed in Discord, it's not an error that can arise in normal usage, only when tinkering with the JSON, in which case the user will see the file getting moved, so I think logging at error level is sufficient.